<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>591</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{background:transparent}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>601</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>591</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{background-color: transparent}
#name{font-size:20px; color: rgb(255,255,255)}
QLabel{color: rgb(255,255,255)}
#category,#autor,#label,#label_2,#label_3,#label_4,#label_5,#label_7,#label_9,#label_10{font-size:17px; color:rgb(202, 231, 201);}
#line, #line_2,#line_3,#line_5,#line_6 {
background-color: rgb(255, 255,255);
border:none
}

QListWidget{
	background-color: transparent;
	border: none;
	color: rgb(255,255,255);
}


QScrollBar::Vertical{
	background-color:rgba(85, 170, 127, 0.4);
	border:none;
	width:10px;

	
}
QScrollBar::handle::Vertical{
	background-color: rgba(131, 255, 112, 0.8);
	height:100px;
	min-height: 50px;
}
QScrollBar::add-line::vertical{
	border:none;
	background:none;
}
QScrollBar::sub-line::vertical{
	border:none;
	background:none;
}

QScrollBar::handle::Vertical::hover{
	background-color:rgba(0, 170, 0, 1);
}
QScrollBar::handle::Vertical::pressed{
	background-color:rgba(0, 170, 127,0.8);
}

QScrollBar::add-Page::vertical,QScrollBar::sub-Page::vertical{
	background:none;
}
QScrollBar::up-arrow::vertical,QScrollBar::down-arrow::vertical{
	background:none;
}</string>
   </property>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>401</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="category">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Категория:</string>
    </property>
   </widget>
   <widget class="QLabel" name="category_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Категория</string>
    </property>
   </widget>
   <widget class="QLabel" name="autor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>85</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Автор:</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>401</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line{
background-color: rgb(0, 170, 0);
border:none
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>401</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line{
background-color: rgb(0, 170, 0);
border:none
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="autor_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>имя автора</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вредители</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Болезни</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_insects">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>191</width>
      <height>88</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Вредитель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>вредит</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент fdsfasdfsafsafsdfsafafds</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="list_sicks">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>201</width>
      <height>88</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <item>
     <property name="text">
      <string>Вредитель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>вредит</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="list_parent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Вредитель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>вредит</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент fdsfasdfsafsafsdfsafafds</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="list_children">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Вредитель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>вредит</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент fdsfasdfsafsafsdfsafafds</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Новый элемент</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>в</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Родительские сорта</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Родитель для </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>429</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Плод:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fruit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Характеристики плода:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fruit_feature">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>401</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line{
background-color: rgb(0, 170, 0);
border:none
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Наличие в селекционных фондах:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_foundations">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>отсутствует</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>401</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line{
background-color: rgb(0, 170, 0);
border:none
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Морозоустойчивость:</string>
    </property>
   </widget>
   <widget class="QLabel" name="frost_resistance">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
